agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileSender.java

```diff
@@ -42,36 +40,34 @@
 
   // send persisted telemetries from local disk every 30 seconds.
   private static final long INTERVAL_SECONDS = 30;
-  private static final ScheduledExecutorService scheduledExecutor =
-      Executors.newSingleThreadScheduledExecutor(
-          ThreadPoolUtils.createDaemonThreadFactory(LocalFileLoader.class));
 
   private final LocalFileLoader localFileLoader;
   private final TelemetryPipeline telemetryPipeline;
 
+  private final ScheduledExecutorService scheduledExecutor =
```

Comment:
to enable `shutdown` method which is a feature needed in azure-monitor-otel-exporter since you can shut it down (unlike in the agent where we don't have to worry about this)